extensions/wikia/WikiaLogo/tests/WikiaLogoTest.php

```diff
@@ -63,19 +63,15 @@ public function testGetCentralUrlForLangWhenCentralWikiDoesNotExist()
 
 	/**
 	 * @dataProvider isFandomExposedDataProvider
-	 * @param $wgEnableGlobalNav2016Value
 	 * @param $lang
 	 * @param $expectedResult
 	 */
-	public function testIsFandomExposed( $wgEnableGlobalNav2016Value, $lang, $expectedResult ) {
-		$this->mockGlobalVariable( 'wgEnableGlobalNav2016', $wgEnableGlobalNav2016Value );
+	public function testIsFandomExposed( $lang, $expectedResult ) {
 		$this->assertEquals( ( new WikiaLogoHelper() )->isFandomExposed( $lang ), $expectedResult );
 	}
 
 	public function isFandomExposedDataProvider() {
 		return [
-			[false, WikiaLogoHelper::FANDOM_LANG, false],
-			[false, 'de', false],
 			[true, 'de', false],
```

Comment:
you need to change data format in Data Provider - method is expecting only two arguments and we are sending three